components/chart/tooltip/overview.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Overview
+page_title: Chart for Blazor | Tooltip Overview
+description: Tooltip for the Blazor Chart
+slug: chart-tooltip-overview
+tags: telerik,blazor,chart,tooltip,tooltips
+published: True
+position: 1
+---
+
+# Tooltip for Telerik Blazor Chart
+
+The Telerik Chart provides a tooltip for its data points. They can be set either to a specific `<ChartSeries>` or as a [shared]({%slug chart-tooltip-shared%}) tooltip for all categories.
+
+In this article:
+
+* [Basics](#basics)
+* [Customization](#customization)
+	* [Parameter Settings](#parameter-settings)
+	* [Template](#template)
+
+
+## Basics
+
+You can enable tooltips for the data points by setting the `Visible` parameter of the `<ChartSeriesTooltip>`, nested inside the `<ChartSeries>` tag, to `true`. By default the value of the point will be presented whey hovered over.
+
+>caption Basic configuration to enable the tooltip for the Chart Series
+
+````CSHTML
+@* Enable tooltip for a Chart Series *@
+
+<TelerikChart>
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Column" Name="Series 1" Data="@data1">
+        </ChartSeries>
+        <ChartSeries Type="ChartSeriesType.Column" Name="Series 2" Data="@data2">
+            <ChartSeriesTooltip Visible="true"></ChartSeriesTooltip>
+        </ChartSeries>
+    </ChartSeriesItems>
+
+    <ChartCategoryAxes>
+        <ChartCategoryAxis Categories="@xAxisItems"></ChartCategoryAxis>
+    </ChartCategoryAxes>
+</TelerikChart>
+
+@code {
+    public List<object> data1 = new List<object>() { 1, 2, 3, 4 };
+    public List<object> data2 = new List<object>() { 2, 3, 4, 5 };
+    public string[] xAxisItems = new string[] { "Q1", "Q2", "Q3", "Q4" };
+}
+````
+>caption The result from the code snippet above
+
+![tooltip basic example](images/tooltip-basic-config.png)
+
+
+## Customization
```

Comment:
Check if the root-level tooltip tag has some settings that apply to all series-specific tooltips and add a section about that too if it does